### PR TITLE
Simplify the EIP template

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: eip-template.md [hunk 1]
```diff
@@ -4,54 +4,44 @@ title: <EIP title>
...
 
-<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
 This is the suggested template for new EIPs.
 
 Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`.
 
 The title should be 44 characters or less.
 
 ## Simple Summary
-<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
```
Comment (axic) [inline eip-template.md:22]: If we can agree on removing these extra comments, I'd also propose to get rid of them in the actual EIPS:
```sh
$ grep "<\!--" * |wc -l
     367
```

**Review by lightclient — APPROVED** [review]
LGTM

**Review by MicahZoltu — APPROVED** [review]

**Comment by axic** [comment]
@MicahZoltu @lightclient okay, removed the template comments from all files...

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - EIP 1 is in state Active, not Draft or Last Call
 - EIP 1167 is in state Final, not Draft or Last Call
 - EIP 152 is in state Final, not Draft or Last Call
 - EIP 1538 is in state Abandoned, not Draft or Last Call
 - EIP 1706 is in state Abandoned, not Draft or Last Call
 - EIP 2159 is in state Final, not Draft or Last Call
 - Error checking file EIPS/eip-712.md
 - EIP 908 is in state Abandoned, not Draft or Last Call
 - File eip-template.md is not an EIP
 - EIP 1080 requires approval from one of (bradleat@inkibra.com)
 - EIP 1081 requires approval from one of (@3esmit, mark.beylin@consensys.net, kevin.owocki@consensys.net)
 - EIP 1129 requires approval from one of (@thunderdeliverer)
 - EIP 1178 requires approval from one of (achon@stanford.edu)
 - EIP 1186 requires approval from one of (simon.jentzsch@slock.it, christoph.jentzsch@slock.it)
 - EIP 1271 requires approval from one of (@izqui, @phabc, @abandeali1, @catageek, @shrugs, @frangio)
 - EIP 1285 requires approval from one of (adam@daostack.io, ben@daostack.io)
 - EIP 1295 requires approval from one of (@atlanticcrypto)
 - EIP 1470 requires approval from one of (@thec00n)
 - EIP 1482 requires approval from one of (@maurelian)
 - EIP 1491 requires approval from one of (@cohabo)
 - EIP 1523 requires approval from one of (@christoph2806)
 - EIP 1616 requires approval from one of (@fiiiu, @spalladino, @thegostep, @0age, @elopio)
 - EIP 1620 requires approval from one of (@paulrberg)
 - EIP 1710 requires approval from one of (@brunobar79)
 - EIP 1775 requires approval from one of (@danfinlay, @bunjin)
 - EIP 1829 requires approval from one of (recmo@0x.org)
 - EIP 1895 requires approval from one of (alexandrebelling8@gmail.com)
 - EIP 1922 requires approval from one of (chaitanya.konda@uk.ey.com, @iammichaelconnor, @westlad)
 - EIP 1923 requires approval from one of (chaitanya.konda@uk.ey.com, @iammichaelconnor, @westlad)
 - EIP 1967 requires approval from one of (@spalladino)
 - EIP 2025 requires approval from one of (@madeoftin)
 - EIP 2026 requires approval from one of (@alexeyakhunov)
 - EIP 2027 requires approval from one of (@alexeyakhunov)
 - EIP 2029 requires approval from one of (@alexeyakhunov)
 - EIP 2031 requires approval from one of (@alexeyakhunov)
 - EIP 2035 requires approval from one of (@alexeyakhunov)
 - EIP 2266 requires approval from one of (@sebastianelvis, jiangshan.yu@monash.edu, chris.haoyul@gmail.com)
 - EIP 2309 requires approval from one of (@pizzarob)
 - EIP 2327 requires approval from one of (@mrchico)
 - EIP 2330 requires approval from one of (@spalladino, @dominicletz)
 - EIP 2378 requires approval from one of (@madeoftin)
 - EIP 2458 requires approval from one of (@edsonayllon)
 - EIP 2481 requires approval from one of (christoph@ethereum.org)
 - EIP 2535 requires approval from one of (@mudgen)
 - EIP 2537 requires approval from one of (@shamatar)
 - EIP 2542 requires approval from one of (forshtat1@gmail.com)
 - EIP 2612 requires approval from one of (@mrchico)
 - EIP 2767 requires approval from one of (@mudgen, @zemse)
 - EIP 2844 requires approval from one of (@oed)
 - EIP 2876 requires approval from one of (@junderw)
 - EIP 2936 requires approval from one of (@wjmelements)
 - EIP 747 requires approval from one of (@danfinlay, @estebanmino)

File: EIPS/eip-2266.md [hunk 2]
```diff
@@ -9,25 +9,20 @@ category: ERC
...
 ---
 
-<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
-
 ## Simple Summary
-<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
 
 A standard for token contracts providing Atomic Swap-based American Call Option functionalities.
 
-## Abstarct
```
Comment (axic) [inline EIPS/eip-2266.md:19]: sic

File: EIPS/eip-1.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ eip: 1
 title: EIP Purpose and Guidelines
 status: Active
 type: Meta
-author: Martin Becze <mb@ethereum.org>, Hudson Jameson <hudson@ethereum.org>, and others
+author: Martin Becze <mb@ethereum.org>, Hudson Jameson <hudson@ethereum.org>, et al.
```
Comment (axic) [inline EIPS/eip-1.md:6]: @lightclient sneaking this in here

**Comment by lightclient** [comment]
This is great!

**Review by wjmelements — APPROVED** [review]

**Review by junderw — APPROVED** [review]

**Review by oed — APPROVED** [review]

**Review by MrChico — APPROVED** [review]

**Review by spalladino — APPROVED** [review]

**Review by 3esmit — APPROVED** [review]
Changes in 1081 were just removed comments, therefore approved.

**Review by frangio — APPROVED** [review]

**Review by shamatar — APPROVED** [review]

**Comment by maurelian** [comment]
> EIP 1482 requires approval from one of (@maurelian)

#1482 is already merged, I'm not sure what I need to do?

**Comment by MicahZoltu** [comment]
Sorry for the ping to everyone.  This is an automated cleanup, but the bot doesn't understand that so pinged everyone.  😬

**Comment by ganmaf** [comment]
> _No description provided._

**Review by drumenbassik — COMMENTED** [review]
2939

**Pull request merged**